aws_lambda/aws_lambda.py

```diff
@@ -167,29 +169,16 @@ def pip_install_to_target(path, requirements=False, local_package=None):
         None
     """
     packages = []
-    if not requirements:
-        log.info('Gathering pip packages')
-        pip_major_version = [int(v) for v in pip.__version__.split('.')][0]
-        if pip_major_version >= 10:
-            from pip._internal import operations
-            packages.extend(operations.freeze.freeze())
-        else:
-            packages.extend(pip.operations.freeze.freeze())
-    else:
+    if requirements:
         if os.path.exists(requirements):
             log.info('Gathering requirement from %s' % requirements)
             data = read_file(requirements)
             packages.extend(data.splitlines())
-        elif os.path.exists("requirements.txt"):
-            log.info('Gathering requirement packages')
-            data = read_file("requirements.txt")
-            packages.extend(data.splitlines())
 
     if not packages:
         log.info('No dependency packages installed!')
 
     if local_package is not None:
-        # TODO: actually sdist is probably bettter here...
         packages.append(local_package)
```

Comment:
Might make sense for the `log.info('No dependency packages installed!')` to be moved down here